qri/dsref.py

```diff
@@ -19,7 +19,7 @@ def __str__(self):
 
 
 def parse_ref(s):
-    m = re.match(r'([a-z][a-z0-9_-]*)/([a-z][a-z0-9_-]*)', s)
+    m = re.match(r'([a-z][a-z0-9_-]*)/([a-z][a-z0-9_-]*)$', s)
```

Comment:
This was allowing `"username/name some other stuff"` to pass, though it would ignore the space onwards.